### PR TITLE
docs: fix the incrementalCacheHandlerPath

### PR DESCRIPTION
**Pull request by jyunhanlin**
The `incrementalCacheHandlerPath` have to be under `experimental`.
And the path should use `path` library so that the custom cache handler can be used.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 9eb0d4cde0714f4ff597f85a15b4b1cc8446fa45

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx [hunk 1]
```diff
@@ -6,8 +6,12 @@ description: Configure the Next.js cache used for storing and revalidating data.
 In Next.js, the [default cache handler](/docs/app/building-your-application/data-fetching/caching) uses the filesystem cache. This requires no configuration, however, you can customize the cache handler by using the `incrementalCacheHandlerPath` field in `next.config.js`.
 
 ```js filename="next.config.js"
+const path = require('path')
+
 module.exports = {
-  incrementalCacheHandlerPath: './cache-handler.js',
+  experimental: {
+    incrementalCacheHandlerPath: path.resolve('./cache-handler.js'),
```
Comment (leerob) [inline docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx:13]: Could this be `require.resolve('./cache-handler.js')` instead?

Comment (jyunhanlin) [reply docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx:13]: Yes, it could be

Comment (jyunhanlin) [reply docs/02-app/02-api-reference/05-next-config-js/incrementalCacheHandlerPath.mdx]: updated

**Review by leerob — APPROVED** [review]

**Pull request merged**